src/main/java/ch/uzh/ifi/hase/soprafs22/game/PlayerAdapter.java

```diff
@@ -0,0 +1,33 @@
+package ch.uzh.ifi.hase.soprafs22.game;
+
+import ch.uzh.ifi.hase.soprafs22.game.enums.Team;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class PlayerAdapter {
```

Comment:
strictly speaking the PlayerAdapter should implement some Interface IPlayer, shouldn't it?